substrate/frame/election-provider-multi-block/src/mock/mod.rs

```diff
@@ -438,6 +438,7 @@ impl ExtBuilder {
 
 	pub(crate) fn build_and_execute(self, test: impl FnOnce() -> ()) {
 		let mut ext = self.build();
+		ext.execute_with(|| roll_one());
```

Comment:
starts by rolling one block so that the System events are initialized. Otherwise they won't be catched and don't show up in the tests.